contents/blog/clickhouse-vs-snowflake.md

```diff
@@ -0,0 +1,146 @@
+---
+date: 2023-01-30
+title: 'In-depth: ClickHouse vs Snowflake'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - mathew-pregasen
+featuredImage: ../images/blog/clickhouse-vs-postgres/postgres-vs-clickhouse.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Guides
+  - ClickHouse
+---
+
+Snowflake versus ClickHouse was a straightforward comparison two years ago. Back then, Snowflake was fully managed, expensive, and broadly featured. ClickHouse, meanwhile, was on-prem, open source, and speed-optimized. Two ends of a data warehouse spectrum splitting versatility (Snowflake) and speed (ClickHouse).
+
+If I was to compare the databases to boats, ClickHouse is a jet ski – ultra-fast, but limited. Snowflake, meanwhile, is a shipyard – a slew of (expensive) tools spanning various functions.
+
+![Clickhouse vs Snowflake.png](../images/blog/clickhouse-vs-snowflake/clickhouse-vs-snowflake.png)
+
+But, as I said, that was *then*. Today, Clickhouse has broken ground on features that compete with Snowflake’s managed solution. Likewise, Snowflake has released features that are beginning to tackle ClickHouse’s speed. However, overlap remains nascent; today, both ClickHouse and Snowflake remain *very, very* different.
+
+## Background
+
+Snowflake and ClickHouse are at wildly different stages of growth.
+
+Snowflake, founded in 2012, is a well-funded, publicly-traded decacorn with nearly 4,000 employees.
+
+In contrast, ClickHouse is in relative infancy. While the project originated around 2012 as a Yandex internal project, ClickHouse was only spun out independently in 2016 and funded in 2021. 
+
+Recently, ClickHouse raised a Series B to launch ClickHouse Cloud, its Snowflake-like service.
+
+### What even is Snowflake?
+
+Every engineer and their cousin has heard about Snowflake. Its massive IPO was recorded as the most successful public launch in technology history. “Next Snowflake” translates to “Next Incredible Business” in venture-capital land. And with over $1B in revenue, the hype is deserved. However, there’s also a rather curious phenomenon – relatively speaking, very few engineers appear to be versed in SnowSQL (Snowflake’s SQL dialect).
+
+This contradiction between success and adoption is because Snowflake doesn’t behave like a typical database company. Snowflake is cloud-only, unlike solutions like ClickHouse, Apache Druid, TimescaleDB, and most other OLAP databases. The codebase is neither open source nor locally installable. And sorry, startups, but there is no free tier.
```

Comment:
```suggestion
This contradiction between success and adoption is because Snowflake doesn’t behave like a typical database company. Snowflake is cloud-only, unlike solutions like ClickHouse, [Apache Druid](/blog/clickhouse-vs-druid), TimescaleDB, and most other OLAP databases. The codebase is neither open source nor locally installable. And sorry, startups, but there is no free tier.
```